### DISCUSSION TITLE
End user devices in Netbox (Evoko)

### DISCUSSION THREAD

**Discussion by PjotrTsjaadajev** [netbox-community/devicetype-library#1680] [Q&A] [answered]
Dear community,

I recently started working with Netbox and I have a few issues and one of them is adding links to end user devices that I can't add to devices/device type.
There is a stack of 8 switches that are the connecting Evoko systems to the network. Evoko is a room booking system for meetings within offices. This type of device does not exist in the dropdown menu at device types. 

Are there any other ways/solutions to still setyp a cable link between my switches and the Evoko systems?

**Comment by DanSheps** [comment, marked as answer]
> This type of device does not exist in the dropdown menu at device types

You (or your netbox Administrator) would have to create this Device Type.

**Reply by PjotrTsjaadajev** [reply]
I had the problem that the manufacturer wouldn't show up in the list. I wasn't aware that I could add them as well, completely missed that. Thanks for your reaction anyway, made me look again and spot the manufacturer option:)
